package.json

```diff
@@ -26,17 +26,19 @@
     "coveralls": "~2.11.6",
     "eslint": "~1.10.3",
     "eslint-config-cp": "transcovo/eslint-config-cp#1.1.0",
+    "express": "^4.14.0",
```

Comment:
why ? the logic is the same as for runtime dependencies IMHO even if the impact is smaller